samples/automl/automlVisionModel.js

```diff
@@ -0,0 +1,607 @@
+/**
+ * Copyright 2018, Google, LLC.
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * This application demonstrates how to perform basic operations on dataset
+ * with the Google AutoML Vision API.
+ *
+ * For more information, see the documentation at
+ * https://cloud.google.com/vision/automl/docs/
+ */
+
+`use strict`;
+
+function createModel(
+  projectId,
+  computeRegion,
+  datasetId,
+  modelName,
+  trainBudget
+) {
+  // [START automl_vision_createModel]
+  const automl = require(`@google-cloud/automl`);
+
+  const client = new automl.v1beta1.AutoMlClient();
+
+  /**
+   * TODO(developer): Uncomment the following line before running the sample.
+   */
+  // const projectId = `The GCLOUD_PROJECT string, e.g. "my-gcloud-project"`;
+  // const computeRegion = `region-name, e.g. "us-central1"`;
+  // const datasetId = `Id of the dataset`;
+  // const modelName = `Name of the model, e.g. "myModel"`;
+  // const trainBudget = `Budget for training model, e.g. 50`;

```

Comment:
Two questions:
1) What currency is this in? `USD`? The user's local one?
2) Can we document what happens if budget isn't specified in this comment?